File: dart2/Dart/Dart2LexerBase.dart
// ignore_for_file: non_constant_identifier_names, file_names

import 'package:antlr4/antlr4.dart';

abstract class Dart2LexerBase extends Lexer {
  Dart2LexerBase(CharStream input) : super(input);

  bool CheckNotOpenBrace() => inputStream.LA(1) != "{".codeUnitAt(0);
}

File: dart2/Python3/Dart2LexerBase.py
from typing import TextIO
from antlr4 import *
from antlr4.Token import CommonToken
import sys
from typing import TextIO

class Dart2LexerBase(Lexer):

    def __init__(self, input: InputStream, output: TextIO = sys.stdout):
        super().__init__(input, output)

    def CheckNotOpenBrace(self):
        return self._input.LA(1) != 123;  # '{'


